MockDataNaming.md

```diff
@@ -94,6 +106,10 @@ with the fallback base name
 ```
 POST|-login-|169d720631e603967135cfce10d235e94aac22b87500ea09d1be295f5b300dca
 ```
+and an optional wildcard base name
+```
+POST|-login-|*
```

Comment:
Yes, the wildcard will allow any query (or body) if enabled.  This coarse grained wildcard implementation was the quickest/easiest thing to do for the problem I have (mocking requests with dates in the query).  I'm hoping that the cases where you have similar requests that need wildcards can be differentiated in other ways than the query or body.  (e.g.: with a unique id in the path `GET /users/1234`). 

I think that adding more fine-grained wildcarding capabilities goes against the simple philosophy of this library